netebpfext/net_ebpf_ext_hook_provider.c

```diff
@@ -4,7 +4,7 @@
 #include "ebpf_extension_uuids.h"
 #include "net_ebpf_ext_hook_provider.h"
 
-#define NET_EBPF_EXT_STACK_EXPANSION_SIZE 1024 * 10
+#define NET_EBPF_EXT_STACK_EXPANSION_SIZE 1024 * 16
```

Comment:
I would still prefer 4K for release and 16K for debug.

Reply:
In latest, it is 4K.